src/Bundle/Form/Type/Filter/RangeNumericFilterType.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\GridBundle\Form\Type\Filter;
+
+use Sylius\Component\Grid\Filter\RangeNumericFilter;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\NumberType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class RangeNumericFilterType extends AbstractType
```

Comment:
Either
```suggestion
final class NumericRangeFilterType extends AbstractType
```
or

```suggestion
final class RangeFilterType extends AbstractType
```

@loic425 Hi, wdyt?